test/test_regression_cli.py

```diff
@@ -1,153 +1,136 @@
 import os
-from unittest import TestCase
-from hashlib import md5
 
-import nose
-
-
-# First md5 is for csv, second md5 is for sqlite
-known_md5s = {
-
-    'sqlite': {'AvianBodySize': '72256f681cdce96eba32d4ece270bcb2',
-               'DelMoral2010': '2f936afb990c8f818223b27e1e8d6212',
-               'MoM2003': 'd3bdce86e0fc5888449884dfb0ef4611'},
-
-    'download': {'AvianBodySize': 'dce81ee0f040295cd14c857c18cc3f7e',
-                 'DelMoral2010': '2b6e92b014ae73ea1f0195ecfdf6248d',
-                 'MoM2003': 'b54b80d0d1959bdea0bb8a59b70fa871'},
-
-    'csv': {'AvianBodySize': 'f42702a53e7d99d16e909676f30e5aa8',
-            'DelMoral2010': 'e79d55ac15f1a70a6c7d3ad4e678ec0e',
-            'MoM2003': 'ef0a31c132cfe1c6594739c872f70f54'},
-
-    'mysql': {'AvianBodySize': 'f60ac93d9be4671dbef77da9d10676b8',
-              'DelMoral2010': 'f241fd296130512d4e1029376b58a4ea',
-              'MoM2003': '9728728d72af4c21a2a6e29fec3edb48'},
-
-    'postgres': {'AvianBodySize': '60c252af74d914e3c15fa9af43edefca',
-                 'DelMoral2010': '63d91a972b5ca07bf90900a576286a8a',
-                 'MoM2003': 'a55c8308722c8e20950e0d1e6d9639e6'}
-}
-
-
-def setup_module():
-    """Update retriever scripts and cd to test directory to find data"""
-    os.chdir("./test/")
+import pytest
+from retriever import SCRIPT_LIST
+from retriever.lib.get_opts import parser
+from retriever.lib.tools import getmd5
+from retriever.lib.tools import name_matches
+from retriever.lib.tools import choose_engine
+
+download_md5 = [
+    # ('DelMoral2010', '0'),
+    ('AvianBodySize', 'dce81ee0f040295cd14c857c18cc3f7e'),
+    ('MoM2003', 'b54b80d0d1959bdea0bb8a59b70fa871')
+]
+
+db_md5 = [
+    # ('DelMoral2010', '0'),
+    ('AvianBodySize', '92ab6bddeb94af8126a8c06d660d418f'),
+    ('MoM2003', 'e0d38086326753eff2fd538864ce3664')
+]
+
+csv_md5 = [
+    # ('DelMoral2010', '0'),
+    ('AvianBodySize', 'e49470d9ca0b2e6142e4b06a96e8b4a0'),
+    ('MoM2003', '8b2d159c3727ea0eb46027033a46a58b')
+]
+
+filedb_md5 = [
+    # ('DelMoral2010', '0'),
+    ('AvianBodySize', '47bdcf49c28fad5a444709f33cbc5c20'),
+    ('MoM2003', 'a605380ab4610d54f431e5413ef35c5e')
+]
+
+
+def setup_module(module):
+    """ setup up any state specific to the execution
+        of the given module.
```

Comment:
Let's be specific about what the function is doing here. e.g. """Set working directory for running tests and update the retriever scripts before starting".